src/category_theory/monoidal/unitors.lean

```diff
@@ -112,7 +112,7 @@ lemma cells_10_13 :
       (α_ (𝟙_ C) ((𝟙_ C) ⊗ (𝟙_ C)) (𝟙_ C)).inv =
     ((𝟙 (𝟙_ C)) ⊗ (ρ_ (𝟙_ C)).inv) ⊗ (𝟙 (𝟙_ C)) :=
 begin
- slice_lhs 1 2 { simp, },
+ slice_lhs 1 2 { rw triangle_assoc_comp_right_inv, },
```

Comment:
More or less a non-terminal simp, that bit me when I changed the simp lemmas above.

Reply:
What does `slice_lhs` do?

Reply:
`slice_lhs a b` uses `conv` to zoom in on the LHS, and then zoom in on the a-th through b-th (inclusive!) morphisms in a chain of morphisms being composed.

Reply:
(rewriting by associativity as necessary to get access to those)

Reply:
That's a really cool feature!